src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  width: 200px;
+  height: 408px;
```

Comment:
```suggestion

```
It's bad practice to specify static width and height, the element size should depend on the content